### PR TITLE
update readme adding details for directory structure

### PR DESCRIPTION
**Pull request by MUzairS15**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by MUzairS15** [comment]
@leecalcote

File: README.md [hunk 1]
```diff
@@ -42,3 +42,26 @@ Access Layer5 Cloud at https://meshery.layer5.io
...
 
 Layer5's cloud native application and infrastructure management software enables organizations to expect more from their infrastructure. We embrace _developer_-defined infrastructure. We empower developers to change how they write applications, support _operators_ in rethinking how they run modern infrastructure, and enable _product owners_ to regain full-control over their product portfolio.
+
+### Directory structure
+1. `action-assets`: 
+      1.   Store the snapshots taken as part of pull requests. Each snapshot have 2 versions and are namespaced based on the year and month they were taken, for eg: snapshot against a particular PR created on 10th July 2024 will be stored at
+      a. Light: `action-assets/2024@7/<design-id>-light.png` 
+      b. Dark version: `action-assets/2024@7/<design-id>-dark.png` 
+ 
+      2. `design-assets`: Stores the snapshots taken for catalog items. Every time a catalog item is updated the corresponding snapshot for that catalog also gets updated. We refer the catalog snaphosts from different locations and hence require a permanenet link.
```
Comment (leecalcote) [inline README.md:52]: @MUzairS15, this is only done for published or for published and public items?

File: README.md [hunk 2]
```diff
@@ -42,3 +42,26 @@ Access Layer5 Cloud at https://meshery.layer5.io
...
+2. `assets`: Contains 3 directories
+      1. `badges`: Stores the SVG and PNG representation of badges to be used in emails.
+      2. `meshmap`: Stores the animated gifs, svgs, quicktips etc,
+      3. `organizations`: Stores the custom icons in PNG format namespaced under orgid.
+      ```
+      organizations/<org-uuid>/mobile.png
+      organizations/<org-uuid>/desktop.png
+      ```
+
+3. `email`: Contains html emails templates, svgs/png to be used for emails sent for `layer5-cloud` and other newsletter html templates.
```
Comment (leecalcote) [inline README.md:67]: @captain-Akshay

File: README.md (hunk 1 above)
Comment (leecalcote) [inline README.md:52]:
```suggestion
      2. `design-assets`: Stores the snapshots taken for catalog items. Every time a catalog item is updated, the corresponding snapshot for that catalog item is also updated. Catalog snapshots are referenced rom different systems, which need unauthenticated, public access to static resources, and therefore, require a permanent link.
```

**Comment by leecalcote** [comment]
@MUzairS15 what is this folder for? Delete it, if it's for nothing - https://github.com/layer5labs/meshery-extensions-packages/tree/master/test

**Comment by leecalcote** [comment]
@MUzairS15 please finish this off.

**Comment by leecalcote** [comment]
Don't forget about me...

File: README.md (hunk 1 above)
Comment (MUzairS15) [reply README.md]: catalog items: i.e. only published

**Pull request merged**